hawtio-util/src/test/java/io/hawt/util/FilesTest.java

```diff
@@ -0,0 +1,298 @@
+package io.hawt.util;
+
+import io.hawt.util.Files;
+import org.junit.Rule;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.mockito.Matchers.isA;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+public class FilesTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+  
+  @Test
+  public void constructor() {
+    // Act, creating object to test constructor
```

Comment:
Remove default constructor test

Reply:
But jacoco then says the class isn't fully covered :cry:

Reply:
Removed.